Automattic/pocket-casts-ios
# Bookmarks: Extracts the bookmark list into its own view

In preparation of being used in more places this extracts the bookmark list view out of the `BookmarksPlayerTab` into `BookmarksListView` and moves the episode specific code from `BookmarkListViewModel` into a new subclass `BookmarkEpisodeListViewModel`.

The majority of the changes are already reviewed code. 

## To test
1. Enable the Bookmarks Feature Flag by going to Profile > Cog > Beta Features > bookmarks
2. Open the Full Screen Player
3. Swipe to the Bookmarks
4. ✅ Verify the bookmarks for the episode still display
5. Add a new bookmark
6. ✅ Verify it displays in the list
7. ✅ Verify the multi select options work
8. ✅ Verify the sorting options work

## Checklist
